.env.example

```
@@ -1,12 +1,25 @@
-# Example configuration for Sepolia + Celestia Mocha.
-TENDERMINT_RPC_URL=https://rpc.celestia-mocha.com/
-CHAIN_ID=11155111
-RPC_URL=https://ethereum-sepolia.publicnode.com/
-CONTRACT_ADDRESS=
-# Key for relaying to the contract.
+# ================= Ethereum
+
+RPC_URL=
 PRIVATE_KEY=
+CONTRACT_ADDRESS=
+
+# ================= Tendermint
+
+TENDERMINT_RPC_URL=
+TENDERMINT_GRPC_URL=
+TENDERMINT_GRPC_BASIC_AUTH=
+
+# ================= SP1
 
 # If you're using the Succinct network, set SP1_PROVER to "network". Otherwise, set it to "local" or "mock".
-SP1_PROVER=
+SP1_PROVER=local
```

We won't use local I assume

```suggestion
SP1_PROVER=
```